docs/destinations/warehouses-and-lakes/rdb/loading-transformed-data/rdb-loader-configuration-reference/index.md

```diff
@@ -92,7 +93,8 @@ This is a complete list of the options that can be configured:
 | `loadAuthMethod.type`                      | Required if `loadAuthMethod` section is included. Specifies the type of the auth method. The possible values are `NoCreds` and `TempCreds`. <br/><br/>With `NoCreds`, no credentials will be passed to `COPY INTO` statement. Databricks cluster needs to have permission to access transformer output S3 bucket. More information can be found [here](https://docs.databricks.com/administration-guide/cloud-configurations/aws/instance-profiles.html). <br/><br/>With `TempCreds`, temporary credentials will be created for every load operation and these temporary credentials will be passed to `COPY INTO` statement. With this way, Databricks cluster doesn't need permission to access to transformer output S3 bucket. This access will be provided by temporary credentials. |
-| `loadAuthMethod.roleArn`                   | Required if `loadAuthMethod.type` is `TempCreds`. IAM role that is used while creating temporary credentials. This role should allow access to the S3 bucket the transformer will write data to, with the following permissions: `s3:GetObject*`, `s3:ListBucket`, and `s3:GetBucketLocation`. |
+| `loadAuthMethod.roleArn`                   | Required if `loadAuthMethod.type` is `TempCreds`. IAM role that is used while creating temporary credentials. This role should allow access to the S3 bucket the transformer will write data to, with the following permissions: `s3:GetObject*`, `s3:ListBucket`, and `s3:GetBucketLocation`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| `logLevel`                                 | Optional. The default value is 3. Specifies JDBC driver log level. <br/><br/> 0 - Disable all logging.<br/><br/> 1 - Log severe error events that lead the driver to abort.<br/><br/> 2 - Log error events that might allow the driver to continue  running.<br/><br/> 3 - Log events that might result in an error if action is not taken. (default)<br/><br/> 4 - Log general information that describes the progress of the  driver.<br/><br/> 5 - Log detailed information that is useful for debugging the driver.<br/><br/> 6 - Log all driver activity.                                                                                                                                                                                                                            |
```

Comment:
```suggestion
| `logLevel` (since 5.4.0)                                | Optional. The default value is 3. Specifies JDBC driver log level. <br/><br/> 0 - Disable all logging.<br/><br/> 1 - Log severe error events that lead the driver to abort.<br/><br/> 2 - Log error events that might allow the driver to continue  running.<br/><br/> 3 - Log events that might result in an error if action is not taken. (default)<br/><br/> 4 - Log general information that describes the progress of the  driver.<br/><br/> 5 - Log detailed information that is useful for debugging the driver.<br/><br/> 6 - Log all driver activity.                                                                                                                                                                                                                            |
```